snomed/com.b2international.snowowl.validation.snomed/src/main/resources/scripts/ruleSnomedCommon1.groovy

```diff
@@ -20,30 +26,33 @@ Iterable<Hits<String>> inactiveConceptBatches = searcher.scroll(Query.select(Str
 		.limit(10_000)
 		.build())
 
-List<ComponentIdentifier> issueDetails = Lists.newArrayList()
+List<ComponentIdentifier> issues = Lists.newArrayList()
 
 inactiveConceptBatches.each({ conceptBatch ->
 	List<String> inactiveConceptIds = conceptBatch.getHits()
-
-	Expression invalidRelationshipExpression = Expressions.builder()
+	
+	ExpressionBuilder invalidRelationshipExpression = Expressions.builder()
 			.filter(SnomedRelationshipIndexEntry.Expressions.active())
 			.should(SnomedRelationshipIndexEntry.Expressions.sourceIds(inactiveConceptIds))
 			.should(SnomedRelationshipIndexEntry.Expressions.typeIds(inactiveConceptIds))
 			.should(SnomedRelationshipIndexEntry.Expressions.destinationIds(inactiveConceptIds))
-			.build()
-
+			
+	if (params.containsKey(EffectiveTimes.UNSET_EFFECTIVE_TIME_LABEL)) {
```

Comment:
Use a different parameter name than `EffectiveTimes.UNSET_EFFECTIVE_TIME_LABEL` for Unpublished only validation